_posts/2023-10-11-14.0.0-release.md

```diff
@@ -0,0 +1,132 @@
+over 3 months of development work and includes [**XXX resolved issues**][1]
+from [**YYY distinct contributors**][2]. See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 13.0.0 release, Metehan Yildirim and Oleks V. have been invited to be committers.
+
+Thanks for your contributions and participation in the project!
+
+## Columnar Format Notes
+
+A `VariableShapeTensorType` was added to the Arrow specification as a canonical extension type. ([GH-24868](https://github.com/apache/arrow/issues/24868)).
+
+Motivated by recent innovations in DuckDB and Meta's Velox engine, new "view" data types were added to the Arrow columnar format spec. 
+
+* 16-byte StringView and BinaryView data type which enables better buffer reuse, faster "false" string comparisons (due to maintaining a prefix) and short string inlining. ([GH-35627](https://github.com/apache/arrow/issues/35627)).
+* ListView and LargeListView types for more performant "out-of-order" building and processing of lists and better buffer reuse ([GH-37876](https://github.com/apache/arrow/issues/37876)).
+
+## Arrow Flight RPC notes
+
+A new RPC method was added to allow polling for completion in long-running queries as an alternative to the blocking GetFlightInfo call ([GH-36155](https://github.com/apache/arrow/issues/36155)). Also, `app_metadata` was added to `FlightInfo` and `FlightEndpoint` ([GH-37635](https://github.com/apache/arrow/issues/37635)).
+
+In C++ and Python, an experimental asynchronous GetFlightInfo call was added to the client-side API ([GH-36512](https://github.com/apache/arrow/issues/36512)). `ServerCallContext` now exposes conveniences to send headers/trailers without having to use middleware ([GH-36952](https://github.com/apache/arrow/issues/36952)). The implementation was fixed to not reject unknown field tags to enable interoperability with future versions of Flight that could add new fields ([GH-36975](https://github.com/apache/arrow/issues/36975)). The CMake configuration was fixed to correctly require linking to Arrow Flight RPC when using Arrow Flight SQL ([GH-37406](https://github.com/apache/arrow/issues/37406)). 
+
+In Go, the underlying generated Protobuf code is now exposed for easier low-level integrations with Flight ([GH-36893](https://github.com/apache/arrow/issues/36893)). 
+
+In Java, the stateful "login" authentication APIs using the Handshake RPC are deprecated; it will not be removed, but it should not be used unless you specifically want the old behavior ([GH-37722](https://github.com/apache/arrow/issues/37722)). Utilities were added to help implement basic Flight SQL services for unit testing ([GH-37795](https://github.com/apache/arrow/issues/37795)).
+
+## C++ notes
```

Comment:
```suggestion
## C++ notes

Experimental APIs for exporting and importing non-CPU arrays using the C Device Data Interface
have been added (GH-36488), together with an experimental API for device synchronization
(GH-36103).

Initial compatibility with Emscripten without threading support has been added (GH-35176).

### Compute layer

New compute functions:
* a `cumulative_mean` function on numeric data (GH-36931);

Improved compute functions:
* rounding functions now work natively on integer inputs instead of casting them to floats (GH-35273);
* the `divide` function now supports duration inputs (GH-36789);
* `take` and `filter` now support sparse unions in addition to dense unions (GH-36905);
* `if_else`, `coalesce`, `choose` and `case_when` now support duration inputs (GH-37028);
* casting between fixed-size lists and variable-size lists is now supported (GH-20086);
* casting from strings to dates is now supported (GH-37411);
* `mean` on integer inputs now uses a floating-point representation for its intermediate sum,
  avoiding integer overflow on large inputs (GH-34909);

### Datasets

Support for writing encrypted Parquet datasets has been added (GH-29238).

### Gandiva

Gandiva now supports linking dynamically to LLVM on non-Windows platforms (GH-37410).
Previously, Gandiva would always link LLVM statically into `libgandiva`.

### Parquet

RLE is used by default when encoding boolean values if v2 data pages are enabled
(GH-36882).

Page indexes can now be encrypted as per the specification (GH-34950).

A bug in the DELTA_BINARY_PACKED encoder leading to suboptimal column sizes was fixed (GH-37939).

### Substrait

It is now possible to serialize and deserialize individual expressions using Substrait,
not only full query plans (GH-33985).

### Miscellaneous

A new `CodecOptions` class allows customizing compression parameters per-codec (GH-35287).

The environment variable `AWS_ENDPOINT_URL` is now respected when resolving S3 URIs (GH-36770).

Recursively listing S3 filesystem trees should now issue less requests,
leading to improved performance (GH-34213).

Comparing a `ChunkedArray` to itself now behaves correctly with NaN values (GH-37515).

The use of BMI2 instructions on x86 was incorrectly guarded. Those instructions
could be executed on platforms without BMI2 support, leading to crashes (GH-37017).

```